v2.0/upgrade-cockroach-version.md

```diff
@@ -162,7 +162,7 @@ For each node in your cluster, complete the following steps.
 
 After upgrading all nodes in the cluster, monitor the cluster's stability and performance for at least one day.
 
-{{site.data.alerts.callout_danger}}During this phase, avoid using any new v2.0 features. Doing so will prevent you from being able to perform a rolling downgrade to v1.1, if necessary.{{site.data.alerts.end}}
+{{site.data.alerts.callout_danger}}During this phase, avoid using any new v2.0 features. Doing so may prevent you from being able to perform a rolling downgrade to v1.1, if necessary.<br><br>.Also, it is not possible to run enterprise [`BACKUP`](backup.html) and [`RESTORE`](restore.html) jobs during this phase.{{site.data.alerts.end}}
```

Comment:
This is now breaking the site.data.alerts.end include for some reason. Check the generated site: http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/45635dc89ff5fb3b7aaee88917928b2c62ad52ab/stable/upgrade-cockroach-version.html#step-5-finalize-or-revert-the-upgrade

Reply:
Done.